### PR TITLE
BUGFIX: Support hubert models that don't have conv_pos_batch_norm configured

### PR DESCRIPTION
**Pull request by igordertigor**
# What does this PR do?

In #34389 , introduced a new configuration flag `conv_pos_batch_norm` that allows using batch norm in the positional conv embeddings. Unfortunately, this new configuration flag is not necessarily present in older hubert pretrained models. By allowing the configuration flag to be absent (and then be treated as the default `False`), these older models work again.

I found this issue with the "m-a-p/MERT-v0" artifact.

- [x] I confirm that this is not a pure code agent PR.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

The following people will likely be quite familiar with the code:
- @gallilmaimon submitted the PR that introduced the issue
- @ylacombe merged the PR that introduced the issue and acted as reviewer on that PR

### CODE REVIEW

**Review by Rocketknight1 — APPROVED** [review]
Yeah, happy to approve this one! The fix should have no impact unless the attribute is missing, in which case the code would crash anyway, so it's not like this should be able to cause any regressions
